### ISSUE TITLE
Support for tables in gherkin

### ISSUE THREAD

**Issue by SamuelMV** [meadsteve/white-bread#63]
Hello, we have started to use white-bread for acceptance tests this week and we have some features (written long time ago) that use tables, for example:

    Scenario: Send message to external service
        Given the customer "Joe" that have an account with id "123456"
        When the customer send a message with the following data
            | origin id | destination id | destination name  | title                                   |
            | 123456  | 9789797         | Harry Potter         | Hello from the other side |
        Then the message is sent to the service bus

But it seems that white-bread doesn't support this kind of gherkin, is this correct? Are you planning to add this feature?

Thanks.

**Comment by SamuelMV** [comment]
Ok, we have found by chance that there is a table_data in the map with this information. Maybe is something that should be described in the docs or examples. Although we are quite new to Elixir, we'll try to help with this ;-)

**Comment by meadsteve** [comment]
@SamuelMV You're right I've somehow completely failed to document this in the readme!

**Comment by pgore** [comment]
@maedsteve would it be possible to have the documentation updated in the near future?
